pyproject.toml

```diff
@@ -20,17 +20,18 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Topic :: Internet :: WWW/HTTP :: Dynamic Content",
     "Topic :: Software Development :: Libraries :: Application Frameworks",
 ]
 requires-python = ">=3.9"
 dependencies = [
     "Flask>=3.0.0,<4.0",
-    "typeguard==4.4.0",
+    "typeguard~=4.4.0",
```

Comment:
Specify the exact version for stability. For example, the update from version 4.3.0 to 4.4.0 of the Typeguard library addressed an issue with list interpretation (see [here](https://github.com/agronholm/typeguard/releases/tag/4.4.0)), which subsequently caused tests to break (see [here](https://github.com/cenobites/flask-jsonrpc/commit/e5b883626986a3d977926b2c2b9f0cb11dacfdda#diff-4e27a385af746f0388192a4a4af0acceca0a76a57f5398ed2119127e97ecbab2L218)).

Reply:
For the all dependencies.